core/src/main/java/org/apache/iceberg/rest/responses/LoadTableResponse.java

```diff
@@ -53,6 +53,8 @@ private LoadTableResponse(String metadataLocation, TableMetadata metadata, Map<S
 
   @Override
   public void validate() {
+    Preconditions.checkNotNull(metadata, "Invalid metadata: null");
+    Preconditions.checkNotNull(metadataLocation, "Invalid metadata location: null");
```

Comment:
Since `validate` is for ensuring that all requirements are met from results received externally, it makes sense to have the `metadata-location` check here.

Reply:
Nevermind. We've determined that `metadata-location` is not a required field. See the PR description for explanation.